source/custom/rules/loaded_kmod_rule.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rules
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+// Rule that matches on loaded kernel modules in the system
+type LoadedKModRule struct {
+}
+
+func (r LoadedKModRule) Match(kmods []string) (bool, error) {
+	log.Printf("Loaded Kernel Modules Rule, Match() on: %v", kmods)
+	loadedModules, err := r.getLoadedModules()
+	if err != nil {
+		return false, fmt.Errorf("failed to get loaded kernel modules. %s", err.Error())
+	}
+	for _, kmod := range kmods {
+		kmodFound := false
+		for _, loadedMod := range loadedModules {
+			if kmod == loadedMod {
+				kmodFound = true
+			}
+		}
+		if !kmodFound {
+			return false, nil
+		}
+	}
+	return true, nil
+}
+
+func (r LoadedKModRule) getLoadedModules() ([]string, error) {
+	out, err := exec.Command("lsmod").Output()
```

Comment:
How about reading directly from `/proc/modules`

Reply:
yes, the implementation was changed to exactly that, unfortunately i amended the wrong commit.